GUI/CKAN-GUI.csproj

```diff
@@ -233,7 +239,7 @@
       <SubType>Component</SubType>
     </Compile>
     <Compile Include="Controls\TriStateToggle.cs">
-      <SubType>Component</SubType>
+      <SubType>UserControl</SubType>
```

Comment:
Was this change intentional? This file doesn't seem to be involved in the other code changes. If Visual Studio did this automatically, maybe I need to investigate whether something is set up wrong in this control.

Reply:
I think I did by mistake